cypress/support/pages/account/pageAccount.js

```diff
@@ -1,31 +1,129 @@
 'use strict'
 
 class Account {
-
-  static #initialsButton = '[data-testid="initials"]';
+  static #initialsButton = '[data-testid="initials"]'
   static #settingsButton = '[data-testid="settings"]'
-
   static #email = '[data-testid="user_email"]'
   static #emailReadOnly = '[data-testid="user_email_readonly"]'
+  static #languageMenu = '#language-menu'
+  static #selectedLanguage = '#selected-language'
+  static #listOfAllLanguages = '.dropdown-submenu >  ul a'
```

Comment:
here data-testid should be assign either in the <class> or in <a>, depending on the DOM

Reply:
data test ids are present on these anchor tags but they are different and not unique. i.e. 
<img width="792" alt="Screenshot 2023-02-14 at 23 43 10" src="https://user-images.githubusercontent.com/97288209/218880144-d6773ad1-2878-4d7c-ad77-7250a8453aa0.png">
Can't be done without unique id here.